charts/matrixone-operator/templates/crds/core.matrixorigin.io_matrixoneclusters.yaml

```diff
@@ -683,6 +678,16 @@ spec:
                   - type
                   type: object
                 type: array
+              credentialRef:
```

Comment:
add an example to `examples/mo-cluster.yaml`? I'm not sure if it is a default config.

Reply:
This is one of the status can will only be populated by the operator after the cluster is ready